src/components/SearchBar/SearchBar.js

```diff
@@ -1,28 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './SearchBar.css';
 import Clickable from '../Clickable/Clickable';
 import line from './line.svg';
 import search from './search.svg';
+import TypingAnimation from '../shared/TypingAnimation';
 
 //TODO: implement search bar query cycling animation
 
 const SearchBar = props => {
+    const [showPlaceholder, setShowPlaceholder] = useState(true);
+    const { value, onSubmit, onChange, onKeyDown } = props;
```

Comment:
This isn't currently the convention for props in our project, but honestly it probably should be. Maybe we can look into refactoring other components to utilize this (outside scope of this PR though).